From 83e08eeed782ba5f0b63f10b27829d031aec74d1 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Sat, 11 Dec 2021 21:22:44 +0100
Subject: [PATCH] remove dependency on sentry

This is unused and breaks ppc64 because it downlaods binaries.
---
 package.json      |  1 -
 webpack.config.js | 11 -----------
 2 files changed, 12 deletions(-)

diff --git a/package.json b/package.json
index 6d740b00b..e94240911 100644
--- a/package.json
+++ b/package.json
@@ -87,7 +87,6 @@
         "@babel/runtime": "^7.12.5",
         "@casualbot/jest-sonar-reporter": "^2.2.5",
         "@principalstudio/html-webpack-inject-preload": "^1.2.7",
-        "@sentry/webpack-plugin": "^1.18.1",
         "@svgr/webpack": "^5.5.0",
         "@testing-library/react": "^12.1.5",
         "@types/flux": "^3.1.9",
diff --git a/webpack.config.js b/webpack.config.js
index 28d60af8a..0018dcdd4 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -8,7 +8,6 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const HtmlWebpackInjectPreload = require("@principalstudio/html-webpack-inject-preload");
-const SentryCliPlugin = require("@sentry/webpack-plugin");
 const crypto = require("crypto");
 
 // XXX: mangle Crypto::createHash to replace md4 with sha256, output.hashFunction is insufficient as multiple bits
@@ -656,16 +655,6 @@ module.exports = (env, argv) => {
                 files: [{ match: /.*Inter.*\.woff2$/ }],
             }),
 
-            // upload to sentry if sentry env is present
-            process.env.SENTRY_DSN &&
-                new SentryCliPlugin({
-                    release: process.env.VERSION,
-                    include: "./webapp/bundles",
-                    errorHandler: (err, invokeErr, compilation) => {
-                        compilation.warnings.push("Sentry CLI Plugin: " + err.message);
-                        console.log(`::warning title=Sentry error::${err.message}`);
-                    },
-                }),
             new webpack.EnvironmentPlugin(["VERSION"]),
         ].filter(Boolean),
 
-- 
2.39.2

